packages/bpk-storybook-utils/src/BpkStorybookUtils.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+/* eslint-disable global-require, no-console, import/no-mutable-exports */
+let action = (...args: Array<any>) => {
+  return () => console.log(args);
+};
```

Comment:
Not sure if this would work, but maybe worth a try for simplification purposes?

```suggestion
let action = console.log
```

Reply:
No because the way it's used is `onClick={action('...')}
If we use `let action = console.log` then the logging will happen when the component is rendered, not when `onClick` happens